### PR TITLE
Events page static html ready

### LINKED ISSUE #1: Create journey view
Create a static view for journeys using journeys API. Something like this image -

Image of events/journey view

### PR DESCRIPTION
**Pull request by ivinjose**
fixes #1

### CODE REVIEW

File: src/views/Events/Events.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import "./Events.css";
+import data from "../../dummy-apis/events";
+
+class Events extends Component {
+    render() {
+        console.log(data);
```
Comment (cijokb) [inline src/views/Events/Events.js]: Remove the console.log

File: src/views/Events/Events.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                <h4>Your Events</h4>
+                <table className="table-events">
+                    {data.events.map((event) => {
+                        return (
+                            <tr className="event">
+                                <td className="date">{event.date}</td>
+                                <td className="name">{event.name}</td>
+                            </tr>
+                        );
+                        console.log("event", event);
```
Comment (cijokb) [inline src/views/Events/Events.js]: Remove the console.log

Comment (ivinjose) [reply src/views/Events/Events.js]: done

File: src/views/Events/Events.js (hunk 1 above)
Comment (ivinjose) [reply src/views/Events/Events.js]: done

**Pull request merged**